package.json

```diff
@@ -69,6 +69,7 @@
     "redux": "^3.7.2",
     "redux-mock-store": "^1.5.1",
     "redux-saga": "^1.0.0",
+    "redux-saga-test-plan": "^4.0.0-beta.3",
```

Comment:
Can this be saved as dev dependencies?

`npm install --save-dev redux-saga-test-plan`

Reply:
Requested changes are made.